aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server/README.md

```diff
@@ -101,19 +101,22 @@ mvn spring-boot:run
 ```
 
 ### Access the Web API
-We could use Postman to simulate a Web APP to send a request to a Web API.
 
 ```http request
```

Comment:
`http request` cannot be recognized in my side.

How about doing like this:

![image](https://user-images.githubusercontent.com/13167207/164585988-197f2e78-b2ce-4859-bc86-c1403f9da511.png)